### ISSUE TITLE
Wrong path of css file

### ISSUE THREAD

**Issue by piotrpog** [ryssbowh/bootstrap-theme#2]
Theme css file path is rendered like this - without base url of site. This makes it stop working when website is in subfolder on server.

`/themes/bootstrap-theme/app.36e1520e.css?v=1659164915`

JS file paths are rendered correctly:

`http://localhost/subfolder/web/cpresources/84d1bc29/bootstrap.bundle.min.js?v=1659164911`

Possibly related to https://github.com/ryssbowh/craft-themes/issues/10

**Comment by ryssbowh** [comment]
Hi,
Which version of Craft and bootstrap theme are you using ?
This is working for me for bootstrap 2.0.4 where https://github.com/ryssbowh/craft-themes/issues/10 was fixed

**Comment by piotrpog** [comment]
System info:

Application Info
PHP version	8.1.2
OS version	WINNT 10.0
Database driver & version	MySQL 5.5.5
Image driver & version	GD 8.1.2
Craft edition & version	Craft Solo 4.2.0.2
Yii version	2.0.45
Twig version	v3.3.10
Guzzle version	7.4.5
Plugins
Bootstrap Theme	2.0.0
Themes	4.2.2

For some reason bootstrap theme is at 2.0.0 and 2.0.4 does not show up in "updates".

**Comment by piotrpog** [comment]
Just installed fresh install with themes plugin and bootstrap theme. Latest version was installed, but now when i open entry witth `themed_page` template, i get error:

```
An Error occurred while handling another error:
ScssPhp\ScssPhp\Exception\CompilerException: `resources-settings.scss` file not found for @import: resources.scss on line 1, at column 0
Call Stack:
#0 import resources.scss app.scss on line 5 in test_themes\vendor\scssphp\scssphp\src\Compiler.php:6102
Stack trace:
```

**Comment by piotrpog** [comment]
Little update - this error showed up only when themes plugin was set to lite edition.
Css file now loads correctly: `/test_themes2/web/themes/bootstrap-theme/app.36e1520e.css?v=1660484596`. But is there any reason why domain name is not included, like with JS files?

**Comment by ryssbowh** [comment]
Hi,
Where are the domain name not included ? In the asset paths in the html ?

**Comment by piotrpog** [comment]
@ryssbowh yes, in css asset path. Its not like it does not work, just wondering why js assets has domain name and css do not.
